src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -722,6 +726,57 @@ public void onFileChange(File file) {
         return dockerfileLines;
     }
 
+    private List<String> removeCopyLines(List<String> dockerfileLines, String buildContext) throws PluginExecutionException {
+        List<String> copyLines = new ArrayList<String>();
+        for (String line : dockerfileLines) {
+            // Remove white space from the beginning and end of the line
+            String trimLine = line.trim();
+            if (!trimLine.startsWith("#")) {
+                // Break the Dockerfile line down into segments based on any amount of whitespace.
+                // The command must be to the left of any comments.
+                String[] cmdSegments = trimLine.split("#")[0].split("\\s+");
+                // If the line starts with COPY
+                if (cmdSegments[0].equalsIgnoreCase("COPY")) {
+                    String src = cmdSegments[cmdSegments.length - 2];
+                    String dest = cmdSegments[cmdSegments.length - 1];
```

Comment:
Validate the length of cmdSegments, otherwise this may hit an array index out of bounds exception.

Reply:
And if the length is not >=3? Do we print an error and throw an Exception?

Reply:
Yes, since COPY requires src and dest